libbeat/docs/shared-kibana-config.asciidoc

```diff
@@ -10,14 +10,15 @@
 //////////////////////////////////////////////////////////////////////////
 
 [[setup-kibana-endpoint]]
-== Configure the Kibana endpoint
+== Configure the {kib} endpoint
 
 ++++
-<titleabbrev>Kibana endpoint</titleabbrev>
+<titleabbrev>{kib} endpoint</titleabbrev>
 ++++
 
-Starting with {beatname_uc} 6.0.0, the Kibana dashboards are loaded into Kibana
-via the Kibana API. This requires a Kibana endpoint configuration.
+Starting with {beatname_uc} 6.0.0, the {kib} dashboards are loaded into {kib}
```

Comment:
6.0.0 is EOL, so I think it's safe to remove this warning

```suggestion
{kib} dashboards are loaded into {kib}
```

Reply:
Thanks!